doc/tutorial/climada_engine_Impact.ipynb

```diff
@@ -68,7 +68,7 @@
     "| event_id |list(int)| id (>0) of each hazard event (Hazard.event_id)|\n",
     "| event_name |(list(str))| name of each event (Hazard.event_name)|\n",
     "| date |np.array| date of events (Hazard.date)|\n",
-    "| coord_exp |np.array| exposures coordinates [lat, lon] (in degrees) (Exposure.gdf.latidues, Exposure.gdf.longitude)|\n",
+    "| coord_exp |np.array| exposures coordinates [lat, lon] (in degrees) (Exposure.gdf['latitudes'], Exposure.gdf['longitude'])|\n",
```

Comment:
I guess it should be 'latitude' here instead of 'latitudes'?

Reply:
Done, good catch.